tutorials/other/ssh_keys.md

```diff
@@ -0,0 +1,91 @@
+Copyright 2015 [Nicky Semenza](http://nickysemenza.com)
+
+#SSH Keys
+
+SSH keys are an easy and secure way of authenticating with a server, without the use of a username or password
+
+
+There are two great uses for them (among others):
+
+ * Authenticating with github (password-less pushing)
+ * Authenticating with servers
+ 
+ 
+##How they work
+
+Basically, on the machine you want to connect to other servers with, you need to generate a public key file and a private key file.
+You then give your public key (id_rsa.pub) to the server you want to connect to, and it adds the key to its 'whitelist', known as the authorized_keys file ***never*** give your private key to anyone, treat it like you would with a secure password.
+##Generating Keys
+### (OSX & Linux based Systems)
+open up terminal, and run the command `ssh-keygen`
+It will ask you where you want to save it, and for an optional passphrase & confirmation of passphrase.
```

Comment:
I would add a note here that most people don't use passphrases unless the key will be used for business or they're very concerned about security. 

If you lose your key (likely means you've lost your computer), you can just invalidate that key anywhere its used.